packages/cdai/src/components/IdeContextHeader/_IdeContextHeader.scss

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright IBM Corp. 2020, 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/scss/carbon-settings';
+@import '../../globals/scss/variables';
+
+.#{$ide-prefix}-context-header {
+  width: 100%;
+  padding: $spacing-03 $spacing-05 7px;
+  background: $ui-background;
+  border-bottom: 1px solid $ui-03;
```

Comment:
can we please use a spacing token here?

Reply:
For the 1px or the 7px? I couldn't find a good token to use for 7px that appeased the linter, so I left it as a plain old 7px... happy to use, say, carbon--mini-units(0.875) if that is preferable.

If it's the 1px... I didn't spot that :)

Reply:
@vladbalanescu I've moved the component across to experimental, and also re-worked the CSS there so I could avoid a linter issue. PTAL.

Reply:
`carbon--mini-units` is the preferred support here, see [this example](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/a3cf68f2eb36351c8b551385fcc88468a3b715c4/packages/cdai/src/components/IdeNavigation/Navigation/_IdeNavigation.scss#L47)